vocab/credentials/v2/vocabulary.yml

```diff
@@ -29,6 +29,12 @@ class:
     label: Credential status
     comment: A Credential Status provides enough information to determine the current status of the credential (for example, suspended or revoked). This class serves as a supertype for specific status types.
 
+  - id: ConfidenceMethod
+    label: Confidence method
+    defined_by: https://w3c-ccg.github.io/confidence-method-spec/
+    comment: A Confidence Method provides enough information for a verifier to determine whether the holder can generate a verifiable presentation to increase the verifier's confidence that they are the same entity referenced by the confidence method. This class serves as a supertype for specific confidence method types.
```

Comment:
Because there will be a lot of wordsmithing to make this acceptable to me and then others... I'll bring mine to the [Confidence Method Spec repo](https://github.com/w3c-ccg/confidence-method-spec).
```suggestion
```